test/test.ts

```diff
@@ -427,3 +427,70 @@ test('can open a specific channel twice', async t => {
   const ch2 = await conn.channel(2)
   t.is(ch2, ch)
 })
+
+
+test('set basic flow on channel', async t => {
+  const amqp = new AMQPClient("amqp://127.0.0.1")
+  const conn = await amqp.connect()
+  const ch = await conn.channel()
+  await t.notThrowsAsync(async () =>  await ch.basicFlow(true))
+})
+
+test('can resolve promise on channel', async t => {
+  const amqp = new AMQPClient("amqp://127.0.0.1")
+  const conn = await amqp.connect()
+  const ch = await conn.channel()
+  t.false(ch.resolvePromise())
```

Comment:
resolvePromise should be a private field, what does this really test?

Reply:
That the method returns false if there are no promises to resolve.